lib/azure/armrest/model/base_model.rb

```diff
@@ -189,7 +190,9 @@ class ResourceProvider < BaseModel; end
     class Sku < BaseModel; end
     class Usage < BaseModel; end
 
-    class ResponseHeaders < BaseModel; end
+    class ResponseHeaders < BaseModel
+      undef_method :response_headers
```

Comment:
undefine a method in a subclass that exists in base class is not an object orient way. Is there other way around?